test-case-reporting/src/test_result_record.ts

```diff
@@ -31,6 +31,18 @@ import md5 from 'md5';
 type QueryFunction = (q: QueryBuilder) => QueryBuilder;
 
 /* eslint-disable @typescript-eslint/camelcase */
+
+/**
+ * Creates a Build object from a build row of the build table.
+ * @param dbBuild A build row of the build table
+ */
+function getBuildFromDbBuild({commit_sha, build_number}: DB.Build): Build {
```

Comment:
tbh I think the level of indirection here isn't worth splitting this out, given how simple it is

Reply:
I don't think I agree, but I don't feel strongly about this.

Done removing getBuildFromDbBuild.